packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -67,7 +72,8 @@ class RemoteConfigManager extends EventEmitter {
           service: config.service,
           env: config.env,
           app_version: config.version,
-          extra_services: []
+          extra_services: [],
+          tags: Object.entries(tags).map((pair) => pair.join(':')).join(',')
```

Comment:
no escaping uh ? sounds like a tag with a colon or a comma will break the parsing in the backend one day ?

Reply:
I took a second look at where I copied this logic from (the Python tracer) and I actually copied it incorrectly. I've pushed a fix now. The right encoding seems to be:

Original
```js
{ a: 1, b: 2 }
```

Encoded:
```js
['a:1', 'b:2']
```

This is still not optimal as a colon in either the tag name of value might mess things up. But it's better. I'm not sure why we try to invent our own encoding when JSON could have encoded this for us without a problem 🤷

Reply:
Well, Nuremberg defense it is then :shrug: i'll resolve the comment